client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -88,15 +88,15 @@ const SHORT_LIST_LENGTH = 6;
 const translateCategory = ( { category, translate } ) => {
 	switch ( category ) {
 		case 'popular':
-			return translate( 'Popular', {
+			return translate( 'Top Free Plugins', {
```

Comment:
@paulopmt1 Should we also change the categories (e.g. "popular"), or do those come from the back-end?

Reply:
Good suggestion @allilevine. This `popular` one we would need to change the WPORG too, because [we use it here](https://opengrok.a8c.com/source/xref/calypso/client/my-sites/plugins/plugins-browser/index.jsx?r=e6ef35f4#158) to filter plugins, so I'll skip it. I'm looking now the other ones because I think we can change them locally :)